src/pages/App.tsx

```diff
@@ -61,13 +65,60 @@ function App(): JSX.Element {
     const dispatch = useDispatch<AppDispatch>()
     const [preservedSource, setPreservedSource] = useState('')
     const sendData = useSendAnalyticsData()
+    const [{ chains, connectedChain }] = useSetChain()
+    const [{ wallet }] = useConnectWallet()
+    const connectedAccount = wallet?.accounts[0]
+    const activeChain = chains.find((chain) => chain.id === connectedChain?.id)
+    const chainId = activeChain?.token === 'CELO' ? 42_220 : 1
+    // const [signedMessage, setSignedMessage] = useState('')
+    // const [messageSignature, setMesssageSignature] = useState('')
 
     void useFaucet()
 
     useEffect(() => {
         sendData({ event: 'goto_page', action: `goto_${pathname}` })
     }, [pathname, sendData])
 
+    // useEffect(() => {
+    //     //Test code for adding signature after wallet connection
+    //     if (wallet) {
+    //         const ethersProvider = new ethers.providers.Web3Provider(wallet.provider, 'any')
+
+    //         const signer = ethersProvider.getSigner()
+    //         signer
+    //             .getAddress()
```

Comment:
if this not needed anymore let's remove it

Reply:
comments have been cleaned up, thanks!  Are there any other issues needed to fix before deployment?